internet-speed-testing/src/main/java/com/example/internet_speed_testing/InternetSpeedBuilder.kt

```diff
@@ -25,6 +28,24 @@ class InternetSpeedBuilder(var activity: Activity) {
         startTestDownload()
     }
 
+    // TODO[Restart properly]: Find a proper way to stop both socket
+    fun stop() {
+        this.LIMIT = 0
+        downloadTestSocket.forceStopTask()
+        uploadTestSocket.forceStopTask()
+//        if (this::downloadTestSocket.isInitialized) {
+//            downloadTestSocket.forceStopTask()
+//        }
+//
+//        if (this::uploadTestSocket.isInitialized) {
+//            uploadTestSocket.forceStopTask()
+//        }
+//
+//        progressModel.progressDownload = 0.toFloat()
+//        progressModel.progressTotal = 0.toFloat()
+//        progressModel.progressUpload = 0.toFloat()
+    }
```

Comment:
- remove comment please. It's still show in your repo.

Reply:
I didn't put those comments in. Do you want me to remove it?

![Screenshot from 2019-10-05 14-37-34](https://user-images.githubusercontent.com/36494062/66261181-b8457a00-e77d-11e9-8646-e5f31c42c2d9.png)

Reply:
Removed.

Reply:
Oh!!! sorry for my misunderstanding. Thank you so much for delete it. I'll merge your PR soon.